### DISCUSSION TITLE
Can not search in PDF with size more 2.9 mb

### DISCUSSION THREAD

**Discussion by patsevanton** [scambier/obsidian-omnisearch#372] [General]
Hello! Thanks for obsidian-omnisearch!
I use obsidian 1.5.12 on Ubuntu 22.04. Obisidian installed from deb package.
I installed text extractor
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/45852e79-fb43-4795-8b7a-377a95ab5c77)

I added different PDF
https://www.researchgate.net/publication/306494162_29_MB_PDF - not searched
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/6269c92d-eeb2-43e7-ab34-165a5ed76d2f)
but if search in file - text searched
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/1d507b2b-5bcb-40d8-a0db-40763c37aafa)

**Comment by patsevanton** [comment]
https://www.fincen.gov/sites/default/files/shared/BOI_Small_Compliance_Guide.v1.1-FINAL.pdf 
searched in file
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/9a6c8bf0-2331-4153-87eb-88195acdc3c7)

but not searched global
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/463a43a5-1dea-490a-9477-31cb3db4e1fc)

**Comment by scambier** [comment]
Have you installed and activated Text Extractor? 

(If you did, many PDFs aren't read correctly by the internal library)

**Comment by patsevanton** [comment]
Yes, I installed text extractor
![image](https://github.com/scambier/obsidian-omnisearch/assets/10828883/45852e79-fb43-4795-8b7a-377a95ab5c77)

**Reply by scambier** [reply]
Thanks, so it's probably a known issue with the internal library. See https://github.com/scambier/obsidian-text-extractor/discussions/21
